integration-tests/src/test/resources/config/application.yml

```diff
@@ -74,6 +74,9 @@ testenv:
   zookeeper:
     host: zk-service1
     port: 2181
+tw-tasks:
+  core:
+    assert-status-on-grabbing: true
```

Comment:
would it perhaps be better to run integration test on CI assuming defaults (and have extra tests checking non-default configuration values)?

Reply:
Tough choice, I don't want to have too long test suite to test both ways.

I usually run test suite locally anyway, so that gives us additional confidence. Plus services have test suites.